src/components/VmDetails/cards/SnapshotsCard/SnapshotItem.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { connect } from 'react-redux'
+
+import {
+  Icon,
+  OverlayTrigger,
+  Tooltip,
+} from 'patternfly-react'
+
+import style from './style.css'
+
+import { msg } from '../../../../intl'
+import RestoreConfirmationModal from './RestoreConfirmationModal'
+import DeleteConfirmationModal from '../../../VmModals/DeleteConfirmationModal'
+import SnapshotDetail from './SnapshotDetail'
+import { deleteVmSnapshot } from './actions'
+import { formatDateFromNow } from '../../../../helpers'
+import { getMinimizedString } from '../../../utils'
+import { PendingTaskTypes } from '../../../../reducers/pendingTasks'
+
+const MAX_DESCRIPTION_SIZE = 50
+
+const StatusTooltip = ({ icon, text, id }) => {
+  return <OverlayTrigger overlay={<Tooltip id={id}>{text}</Tooltip>} placement='left' trigger={['hover', 'focus']}>
+    <a>{icon}</a>
+  </OverlayTrigger>
+}
+
+StatusTooltip.propTypes = {
+  icon: PropTypes.node.isRequired,
+  text: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+}
+
+function isSnapshotBeingDeleted (snapshotId, pendingTasks) {
+  return !!pendingTasks.find(task => task.type === PendingTaskTypes.SNAPSHOT_REMOVAL && task.snapshotId === snapshotId)
+}
+
+const SnapshotItem = ({ snapshot, vmId, isEditing, beingDeleted, onSnapshotDelete }) => {
+  let buttons = []
+  let statusIcon = null
+  if (!snapshot.get('isActive')) {
+    buttons.push(<OverlayTrigger
+      overlay={<SnapshotDetail key='detail' snapshot={snapshot} vmId={vmId} />}
+      placement='left'
+      trigger='click'
+      rootClose
+      key='info'
+    >
+      <a><Icon type='pf' name='info' /></a>
```

Comment:
If the `a` is just a tag to style, it can be any other tag (or no tag at all).  Do browsers still complain about `a` tags without an `href`?

Reply:
Not any, but yes, and it can be `a` as well. Browsers don't complain about that.